Platforms/QemuQ35Pkg/QemuFlashFvbServicesRuntimeDxe/FwBlockServiceDxe.c

```diff
@@ -224,26 +224,9 @@ SetPcdFlashNvStorageBaseAddresses (
   VOID
   )
 {
-  RETURN_STATUS  PcdStatus;
-
   //
-  // Set several PCD values to point to flash
+  // Do nothing.

```

Comment:
Aren't all the instances of `SetPcdFlashNvStorageBaseAddresses()` now doing nothing?

Can the caller and functions be entirely removed?